Formula/apache-ctakes.rb

```diff
@@ -0,0 +1,85 @@
+           "#{libexec}/resources/org/apache/ctakes/dictionary/lookup/."
+        mv "#{libexec}/umls/org/apache/ctakes/dictionary/lookup/rxnorm-hsqldb", \
+           "#{libexec}/resources/org/apache/ctakes/dictionary/lookup/."
+        mv "#{libexec}/umls/org/apache/ctakes/dictionary/lookup/rxnorm_index", \
+           "#{libexec}/resources/org/apache/ctakes/dictionary/lookup/."
+        mv "#{libexec}/umls/org/apache/ctakes/dictionary/lookup/umls2011ab", \
+           "#{libexec}/resources/org/apache/ctakes/dictionary/lookup/."
+        mv "#{libexec}/umls/org/apache/ctakes/lvg/data/build.txt", \
+           "#{libexec}/resources/org/apache/ctakes/lvg/data/."
+        mv "#{libexec}/umls/org/apache/ctakes/lvg/data/config/lvg.properties.mysql", \
+           "#{libexec}/resources/org/apache/ctakes/lvg/data/config/."
+        mv "#{libexec}/umls/org/apache/ctakes/lvg/data/config/lvg.properties.org", \
+           "#{libexec}/resources/org/apache/ctakes/lvg/data/config/."
+        remove_dir "#{libexec}/resources/org/apache/ctakes/lvg/data/HSqlDb"
+        mv "#{libexec}/umls/org/apache/ctakes/lvg/data/HSqlDb", \
+           "#{libexec}/resources/org/apache/ctakes/lvg/data/."
+        remove_dir "#{libexec}/resources/org/apache/ctakes/lvg/data/rules"
+        mv "#{libexec}/umls/org/apache/ctakes/lvg/data/rules", \
+           "#{libexec}/resources/org/apache/ctakes/lvg/data/."
+        mv "#{libexec}/umls/org/apache/ctakes/lvg/data/ReadMe.txt", \
+           "#{libexec}/resources/org/apache/ctakes/lvg/data/ReadMe_UMLS.txt"
+        mv "#{libexec}/umls/org/apache/ctakes/lvg/data/tables", \
+           "#{libexec}/resources/org/apache/ctakes/lvg/data/."
+        mv "#{libexec}/umls/org/apache/ctakes/lvg/data/version.txt", \
+           "#{libexec}/resources/org/apache/ctakes/lvg/data/."
+        remove_dir "#{libexec}/umls"
+      end
+    end
+  end
+
+  test do
+    (testpath/"notes.txt").write <<-EOS.undent
+      Dr. Nutritious
+
+      Medical Nutrition Therapy for Hyperlipidemia
+
+      Referral from: Julie Tester, RD, LD, CNSD
+      Phone contact: (555) 555-1212
+      Height: 144 cm   Current Weight: 45 kg   Date of current weight:
+      02-29-2001 Admit Weight:  53 kg   BMI: 18 kg/m2
+      Diet: General
+      Daily Calorie needs (kcals): 1500 calories, assessed as HB + 20%
+      for activity.
+      Daily Protein needs: 40 grams,  assessed as 1.0 g/kg.
+      Pt has been on a 3-day calorie count and has had an average
+      intake of 1100 calories.  She was instructed to drink 2-3 cans
+      of liquid supplement to help promote weight gain.  She agrees
+      with the plan and has my number for further assessment. May want
+      a Resting Metabolic Rate as well. She takes an aspirin a day for
+      knee pain.
+    EOS
+    system "#{bin}/runPiperFile.sh", "-p", "#{libexec}/resources/org/apache/ctakes/examples/pipeline/HelloWorld.piper", "-i", "#{testpath}/notes.txt"
```

Comment:
This is pretty much a no-op that just tests that `ctakes` can find basic resources in its paths. It doesn't output anything.